javascript/javascript2/week3/Clicking Game/Clicking Game Thing.js

```diff
@@ -0,0 +1,96 @@
+const btn = document.getElementById("startBtn");
+let inputTime = document.getElementById('inputTime');
+const timer = document.getElementById("timer");
+const scoreS = document.getElementById("scoreS");
+const scoreL = document.getElementById("scoreL");
+const buttonForS = document.getElementById("right");
+const buttonForL = document.getElementById("left");
+
+
+let startNUmS = 0;
+let startNUmL = 0;
+
+btn.addEventListener("click", () => {
+    let StartingSec = inputTime.value;
+    ended = false;
+    startGame()
+    timer.innerHTML = `${StartingSec} : 00`;
+    const countdown = setInterval(() => {
+        StartingSec--;
+        timer.innerHTML = `${StartingSec} : 00`;
+        if (StartingSec <= 0) {
+            ended = true;
+            clearInterval(countdown);
+            endGame();
+        }
+    }, 1000);
+})
+function startGame() {
+    document.addEventListener("keypress", (event) => {
+        if (event.code !== "KeyS" && event.code !== "KeyL") {
```

Comment:
Instead of using Keypress, it's a better practice to use keydown event. https://developer.mozilla.org/en-US/docs/Web/API/Element/keydown_event